tasks/updatelibrary.js

```diff
@@ -1,17 +1,64 @@
 'use strict';
 
+const fs = require('fs');
 const path = require('path');
+const crypto = require('crypto');
+const existsSync = require('exists-sync');
+function hash (contents) {
+	return crypto.createHash('md5').update(contents).digest('hex');
+}
+
+function constructPolyfill(polyfillPaths) {
+	const polyfills = polyfillPaths.map((polyfill) => fs.readFileSync(polyfill));
+	return polyfills.join('');
+}
+
+const installPolyfill = function(polyfillOutputFolder, module, polyfillSourcePaths) {
+	const polyfillOutputPath = path.join(polyfillOutputFolder, 'polyfill.js');
+	const polyfillAlreadyExists = existsSync(polyfillOutputPath);
+
+	polyfillSourcePaths = polyfillSourcePaths || [''];
+	polyfillSourcePaths = polyfillSourcePaths.map(polyfillPath => path.join(module, polyfillPath))
+	polyfillSourcePaths = polyfillSourcePaths.map(path => require.resolve(path));
+
+	console.log(`Importing polyfill/s from ${polyfillSourcePaths.join(',')}`);
```

Comment:
use `grunt.log` if possible?
Consider starting the comment with our polyfill feature name, I can't see that we're outputting that otherwise.